docs/conf.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python3
+#
+# Copyright 2017 Spotify AB
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import codecs
+import os
+import re
+
+
+def read(*parts):
```

Comment:
Inspired by the [attrs project](https://github.com/python-attrs/attrs/blob/master/docs/conf.py#L8-L28).